docs/Examples/Basic.md

```diff
@@ -59,6 +59,16 @@ extension MyService: TargetType {
             return JSONEncoding.default // Send parameters as JSON in request body
         }
     }
+    var task: Task {
+        switch self {
+        case .zen, .showUser, .showAccounts: // Send no parameters
+            return .requestPlain
+        case .createUser(let firstName, let lastName): // Send parameters in URL
```

Comment:
In fact `URLEncoding.default` sends parameters in URL when possible, not always (see [#ParameterEncoding](https://github.com/Alamofire/Alamofire#parameter-encoding)).